### PR TITLE
add `replicate.models.list()` method for listing public models

### PR DESCRIPTION
**Pull request by zeke**
Yep!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,6 +199,39 @@ const response = await replicate.models.get(model_owner, model_name);
 }
 ```
 
+### `replicate.models.list`
+
```
Comment (zeke) [inline README.md:203]: It seems like we should have a one-liner description here below the heading, but there's no precedent for that in the existing docs. What do you think @mattt ?

Comment (mattt) [reply README.md:203]: I'm all for it!

**Comment by zeke** [comment]
Oh god I just noticed all the unrelated changes here that resulted from my mysterious VSCode setup. 🙈 

See https://github.com/replicate/replicate-javascript/issues/119

**Comment by zeke** [comment]
Cleaned up the littany of lintening and force-pushed a cleaner commit.

**Review by mattt — APPROVED** [review]

**Pull request merged**